src/main/java/uk/gov/hmcts/ccd/domain/service/common/CaseDataService.java

```diff
@@ -110,17 +110,24 @@ private void deduceClassificationForCollectionType(JsonNode dataNode, JsonNode e
             ArrayNode arrayNode = (ArrayNode) fieldNode;
             final FieldType collectionFieldType = caseField.getFieldType().getCollectionFieldType();
             for (JsonNode field : arrayNode) {
+
+                final JsonNode itemClassification = getExistingDataClassificationFromArrayOrEmpty(
+                    existingDataClassificationNode,
+                    field);
+
                 if (COMPLEX_TYPE.equalsIgnoreCase(collectionFieldType.getType())) {
                     deduceDefaultClassifications(
                         field.get(VALUE),
                         // get value of the field with given ID
-                        getExistingDataClassificationFromArrayOrEmpty(existingDataClassificationNode, field),
+                        itemClassification,
                         caseField.getFieldType().getCollectionFieldType().getComplexFields(),
                         fieldIdPrefix + fieldName + FIELD_SEPARATOR);
                 } else {
                     final ObjectNode simpleCollectionItemNode = (ObjectNode) field;
                     // Add `classification` property
-                    deduceClassificationForSimpleType(simpleCollectionItemNode, existingDataClassificationNode, CLASSIFICATION, caseField);
+                    final String newClassification = itemClassification.isTextual() ?
+                        itemClassification.textValue() : getClassificationFromCaseFieldOrEmpty(caseField);
```

Comment:
Not related to your changes but can we change the name of the getClassificationFromCaseFieldOrEmpty to something closer to truth like getClassificationFromCaseFieldOrDefault?